slidez-fe-core/src/common/components/forms/sign-form.scss

```diff
@@ -6,6 +6,7 @@
   flex-direction: column;
   width: 45%;
   margin-right: 10px;
+  font-family: 'Rubik', serif;
```

Comment:
Font should be written in body, as global style, please, don't set it this way
If needed set it in global.scss to all div or what this component is)

Reply:
I mean font-family